contrib/guix/libexec/build.sh

```diff
@@ -368,6 +368,10 @@ mkdir -p "$DISTSRC"
                 ;;
         esac
 
+        # copy over the example bitcoin.conf file. if contrib/devtools/gen-bitcoin-conf.sh
+        # has not been run before buildling, this file will be a stub
+        cp "${DISTSRC}/share/examples/bitcoin.conf" "${DISTNAME}/"
```

Comment:
Install a `bitcoin.conf"` file using Windows installer as well?

Reply:
I'll combine this into https://github.com/bitcoin/bitcoin/pull/25829.